Detections/Corelight/CorelightNetworkServiceScanning.yaml

```diff
@@ -0,0 +1,33 @@
+id: 599570d4-06f8-4939-8e29-95cd003f1abd
+name: Corelight - Network Service Scanning Multiple IPs
+description: |
+  'Identify scanning of services that may be available on the internal network.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated